src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/create_index_pattern_wizard.tsx

```diff
@@ -31,41 +31,68 @@ import { EmptyState } from './components/empty_state';
 
 import { MAX_SEARCH_SIZE } from './constants';
 import { ensureMinimumTime, getIndices } from './lib';
-import { i18n } from '@kbn/i18n';
+import {
+  SavedObjectsClient,
+  IUiSettingsClient,
+  OverlayStart,
+} from '../../../../../../../../core/public';
+import { DataPublicPluginStart } from '../../../../../../../../plugins/data/public';
+import { IndexPatternCreationConfig } from '../../../../../../../../plugins/index_pattern_management/public';
+import { MatchedIndex } from './types';
+
+interface CreateIndexPatternWizardProps {
+  initialQuery: string;
+  services: {
+    indexPatternCreationType: IndexPatternCreationConfig;
```

Comment:
nit: maybe we can move it to new interface

Reply:
Do you mean to create a separate interface for services?

Reply:
yep

Reply:
When we convert all components to react we will add KibanaContextProvider and move dependencies there.